src/common/commonutils/FileUtils.c

```diff
@@ -1608,15 +1623,16 @@ static int FindTextInCommandOutput(const char* command, const char* text, void*
     char* results = NULL;
     int status = 0;
 
-    if ((NULL == command) || (NULL == text))
+    if ((NULL == command) || (NULL == text) || (0 == strlen(command)) || (0 == strlen(text)))
     {
         OsConfigLogError(log, "FindTextInCommandOutput called with invalid argument");
         return EINVAL;
     }
 
-    if (0 == (status = ExecuteCommand(NULL, command, true, false, 0, 0, &results, NULL, log)))
+    // Execute this command with a 60 seconds timeout
+    if (0 == (status = ExecuteCommand(NULL, command, true, false, 0, 60, &results, NULL, log)))
     {
-        if (NULL != strstr(results, text))
+        if ((NULL != results) && (0 < strlen(results)) && (NULL != strstr(results, text)))

```

Comment:
Is the strlen() call needed here? Looks like strstr() should be enough.

Reply:
Don't want to call strstr on empty string as it may falsely return.